### DISCUSSION TITLE
Missing "example custom config" and "Notes" after initial installation

### DISCUSSION THREAD

**Discussion by informatik01** [NvChad/NvChad#2775] [Q&A] [answered]
Hi.

Just tried today installing NvChand and unlike what I saw in multiple tutorials, there were two moments missing:

- 👉 No prompt `"Do you want to install example custom config? (y/N)"`
- 👉 No final "NOTES" after installation process was supposedly finished ⤵️
---
<img width="983" alt="image" src="https://github.com/NvChad/NvChad/assets/11339880/d091046f-c1d9-44d4-acb6-40ed76a3c3be">

---
 
And there is no `custom` folder in the `~/.config/nvim` directory as a result of item (1).

Is it normal now (as of **NvChad version 2.5**) or something's not right?

**Comment by siduck** [comment, marked as answer]
its normal, there's no custom dir in v2.5 , its like any other neovim config now, you  have total control over it

**Reply by informatik01** [reply]
OK, thank you.

**Reply by coder-eb** [reply]
@siduck, Might be a noob question, but, I don't understand how this is better than the previous version which allowed us to customize to our heart's content without touching the core NvChad files.

**Reply by siduck** [reply]
@coder-eb with custom config u had to learn custom syntax + nvim config syntax. with new one you just know the nvim config syntax. 

It gives you full freedom than before, before u couldnt even disable all mappings at once, now you can just not include `require "nvchad.mappings"` in your config! and so on! 

Its just like a regular config and uses nvchad main repo as a plugin and imports its modules, what things are you not able to customize? tell me
